_posts/2019-02-15-release-1.6.4.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "Apache Flink 1.6.4 Released"
+date:   2019-02-15 12:00:00
+categories: news
+---
+
+The Apache Flink community released the fourth bugfix version of the Apache Flink 1.6 series.
+
+This release includes more than 60 fixes and minor improvements for Flink 1.6.3. The list below includes a detailed list of all fixes.
```

Comment:
`60` -> `25` (according to the Jira release notes)?

Reply:
+1